CalibCalorimetry/HcalAlgos/src/HcalTimeSlew.cc

```diff
@@ -27,7 +27,7 @@ double HcalTimeSlew::delay(double fC, ParaSource source, BiasSetting bias, doubl
   }
   else if (source==Data || source==MC){
     if(isHPD) return std::fmin(cap,tspar0[source-1]+tspar1[source-1]*log(fC+tspar2[source-1]));
-    return std::fmin(cap,tspar0_siPM[source-1]+tspar1_siPM[source-1]*log(fC+tspar2_siPM[source-1]));
+    return cap+tspar0_siPM[source-1];
```

Comment:
@slava77 
Only the M3 delay is changed in this PR.

This function is used in the M2 (offline data/MC)
https://github.com/cms-sw/cmssw/blob/CMSSW_9_0_X/CalibCalorimetry/HcalAlgos/src/HcalTimeSlew.cc#L15-L18

This function is used in the M3 (online)
https://github.com/cms-sw/cmssw/blob/CMSSW_9_0_X/CalibCalorimetry/HcalAlgos/src/HcalTimeSlew.cc#L20-L33

This is what is in the simulation
https://github.com/cms-sw/cmssw/blob/CMSSW_9_0_X/SimCalorimetry/HcalSimAlgos/src/HcalTimeSlewSim.cc